src/box/index_def.c

```diff
@@ -48,6 +48,7 @@ const struct index_opts index_opts_default = {
 	/* .run_count_per_level = */ 2,
 	/* .run_size_ratio      = */ 3.5,
 	/* .bloom_fpr           = */ 0.05,
+    /* .lookup_cost_coeff   = */ 0.5,
```

Comment:
This requires discussion.

Reply:
Fixed the indent.
Seems like we should get default value by experimenting with the select queries latency so it approximately equals latency with bloom_fpr = 0.05.

Reply:
Looks like you didn't fix indent.

Reply:
Fixed it (Had to rebase and move it from one commit to another).